src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -389,14 +390,6 @@ export default class MessageContextMenu extends React.Component {
             </MenuItem>
         );
 
-        if (mxEvent.getType() !== mxEvent.getWireType()) {
-            viewClearSourceButton = (
-                <MenuItem className="mx_MessageContextMenu_field" onClick={this.onViewClearSourceClick}>
```

Comment:
We can remove `onViewClearSourceClick` now I assume?

Reply:
You are right, done